src/Analysis/GWASV2/Shared/AttritionTableWrapper/AttritionTable/AttritionTableRow.jsx

```diff
@@ -7,16 +6,16 @@ import {
   fetchConceptStatsByHareSubset,
   fetchConceptStatsByHareSubsetCC,
   queryConfig,
-} from '../wizardEndpoints/cohortMiddlewareApi';
-import BarChart from './ChartIcons/BarChart';
-import EulerDiagram from './ChartIcons/EulerDiagram';
-import './AttritionTable.css';
+} from '../../wizardEndpoints/cohortMiddlewareApi';
+import BarChart from '../ChartIcons/BarChart';
+import EulerDiagram from '../ChartIcons/EulerDiagram';
 
 const AttritionTableRow = ({
-  cohortDefinitionId,
   otherCohortDefinitionId,
+  selectedCohort,
   rowType,
-  rowName,
+  rowObject,
+  outcome,
   covariateSubset,
```

Comment:
the best name for this one is `currentCovariateAndCovariatesFromPrecedingRows`

Reply:
or `currentCovariatePlusCovariatesFromPrecedingRows` 🤔

Reply:
I prefer the first one, plus could imply addition